hs/test-examples/features/exception_return.rsh

```diff
@@ -0,0 +1,27 @@
+'reach 0.1';
+
+const returnsFromCatch = () => {
+  try {
+    throw 5;
+    return false;
+  } catch (e) {
+    return true;
+  }
+}
+
+const returnsFromTry = () => {
+  try {
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
+
+export const main =
+  Reach.App(
+    {},
+    [Participant('A', {})],
+    (A) => {
+      assert(returnsFromCatch());
+      assert(returnsFromTry());
```

Comment:
I think this only works because the try never fails. If you were to instead have written `try { if (x < x) { throw false; } else { return true } } catch (e) { return false }` where `x` was an interact variable, then I believe this assert would fail